server/src/main/java/org/elasticsearch/index/mapper/ParametrizedFieldMapper.java

```diff
@@ -178,6 +180,11 @@ public T getValue() {
             return isSet ? value : defaultValue.get();
         }
 
+        @Override
```

Comment:
Are you thinking of removing the old `getValue` in a follow up?

Reply:
That's the plan, yes